src/Microsoft.AspNetCore.Mvc.TagHelpers/EnvironmentTagHelper.cs

```diff
@@ -30,13 +34,33 @@ public EnvironmentTagHelper(IHostingEnvironment hostingEnvironment)
 
         /// <summary>
         /// A comma separated list of environment names in which the content should be rendered.
+        /// If the current environment is also in the <see cref="Exclude"/>  list, the content will not be rendered.
         /// </summary>
         /// <remarks>
         /// The specified environment names are compared case insensitively to the current value of
         /// <see cref="IHostingEnvironment.EnvironmentName"/>.
         /// </remarks>
         public string Names { get; set; }
 
+        /// <summary>
+        /// A comma separated list of environment names in which the content should be rendered.
+        /// If the current environment is also in the <see cref="Exclude"/> list, the content will not be rendered.
+        /// </summary>
+        /// <remarks>
+        /// The specified environment names are compared case insensitively to the current value of
+        /// <see cref="IHostingEnvironment.EnvironmentName"/>.
+        /// </remarks>
+        public string Include { get; set; }
+
+        /// <summary>
+        /// A comma separated list of environment names in which the content should not be rendered.
```

Comment:
Replace `should` with `will`. In fact, make this change in all the comments (assuming it applies 😄 ).